packages/blade/src/components/Button/BaseButton/StyledBaseButton.native.tsx

```diff
@@ -75,13 +87,27 @@ const _StyledBaseButton: React.ForwardRefRenderFunction<TextInput, StyledBaseBut
     };
   });
 
+  const handleOnPress = (event: GestureResponderEvent): void => {
+    if (href) {
+      void openURL(href);
+    }
+
+    if (onClick) {
+      /*
+      React Native's Pressable's onClick returns a GestureResponderEvent but our types expect a SyntheticEvent.
+      Until we have a way to handle platform specific types, we will have to ignore this TS error.
```

Comment:
We have a way to handle platform specific types, no?

Reply:
ops it was stale comment from BaseLink. Its not valid for BaseButton anyway. Removing it.